lib/modules/blockchain_process/blockchain.js

```diff
@@ -60,7 +60,7 @@ var Blockchain = function(userConfig, clientClass) {
     targetGasLimit: this.userConfig.targetGasLimit || false,
     syncMode: this.userConfig.syncMode || this.userConfig.syncmode,
     verbosity: this.userConfig.verbosity,
-    proxy: this.userConfig.proxy || true
+    proxy: this.userConfig.hasOwnProperty('proxy') ? this.userConfig.proxy : true
```

Comment:
Instead of checking for the property exists, do you think we could have a default value in core/config
That way it could just be:
`proxy: this.userconfig.proxy`

Reply:
I'd vote for that as well.